tests/parallel/test_parallel_enumerable_thread_local.h

```diff
@@ -38,7 +38,7 @@
 
 namespace test_align_parallel
 {
-struct alignas(128) TestValue
+struct TestValue
```

Comment:
why no alignment anymore?

Reply:
Well, it just doesn't make sense. Also not in the context tested.